app/src/androidTest/java/org/mozilla/tv/firefox/ui/ToggleDataCollectionTest.kt

```diff
@@ -58,6 +58,7 @@ class ToggleDataCollectionTest {
         }.exitToOverlay {
         }.openSettings {
             assertDataCollectionButtonState(cachedDataCollectionButtonIsChecked)
+            toggleDataCollectionButton() // For resetting the data on Cloud Test
```

Comment:
I see, so this is for manually "undoing" the data collection click?

Can you make the comment more specific, like // Manually undo data collection button click until Issue #XXX to correctly clear data between tests with Test Orchestrator is resolved